File: service/SurfWebAutofac/Model/Dtos/NewRecords/NewMapDto.cs
namespace Model.Dtos.NewRecords;

public class NewMapDto
{
    public  long Id { get; set; }

    /// <summary>
    ///     名称
    /// </summary>
    public  string Name { get; set; }

    /// <summary>
    ///     难度
    /// </summary>
    public  string Difficulty { get; set; }

    /// <summary>
    ///     地图图片
    /// </summary>
    public  string Img { get; set; }

    /// <summary>
    ///     创建日期
    /// </summary>
    public DateTime CreateTime { get; set; }
}

File: service/Java/SurfWeb/SurfWeb/system/src/main/java/com/surfweb/system/dto/newrecords/NewMapDto.java
package com.surfweb.system.dto.newrecords;

import lombok.Data;

import java.util.Date;

@Data
public class NewMapDto {

    private long id;

    /**
     * 名称
     */
    private String name;

    /**
     * 难度
     */
    private String difficulty;

    /**
     * 地图图片
     */
    private String img;

    /**
     * 创建日期
     */
    private Date createTime;
}
